systemd/systemd.go

```diff
@@ -351,9 +373,16 @@ const (
 	GlobalUserMode
 )
 
+type Kind int
```

Comment:
seeing it in practice I'm not enthusiastic that Kind is a good name for this, but I don't want to block the PR which is blocking the mount-control landing. So let's land this and I will think about naming and propose a small PR later about naming here.

Reply:
Excellent, thanks! :-)